docs/quickstart/bootstrap.rst

```diff
@@ -104,6 +104,11 @@ Bootstrap node.
 
    root@bootstrap $ /srv/scality/metalk8s-|release|/bootstrap.sh
 
+.. warning::
+
+    In case of virtual network (or any network not able to route workload IP
```

Comment:
_"Workload IP addresses"_ is unclear to me, maybe you meant _"cluster-internal IP"_